src/image_service/image_fs_info.rs

```diff
@@ -1,10 +1,10 @@
 use crate::{
     criapi::{ImageFsInfoRequest, ImageFsInfoResponse},
-    image_service::MyImage,
+    service::Service,
```

Comment:
I like the refactoring here but maybe we can pick a more descriptive name than `Service`?

Reply:
Hm, something with GRPC? GrpcServer?

Reply:
@haircommander any thoughts on a good name?

Reply:
I would say CRIService

Reply:
Renamed it to `CRIService`